src/style-sheets/generic/_card.scss

```diff
@@ -1,9 +1,83 @@
 .card-module {
   margin: 10px;
-  min-width: 235px;
-  min-height: 402px;
 
   background-color: $olLTPrimary;
   box-shadow: 0px 4px 4px rgba(60, 129, 137, 0.25);
   border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  width: 324px;
+  height: 180px;
+
+  @include medium('up') {
+    height: 365px;
+    width: 234px;
+    margin-right: 77px;
+    margin-bottom: 100px;
+  }
+  .card-content {
+    display: grid;
+    grid-template-columns: 0.5fr 0.4fr 1fr;
+    grid-template-rows: 0.8fr 0.4fr 1fr;
+    margin-left: 15px;
+    @include medium('up') {
+      margin: 2px;
+      display: flex;
+      flex-direction: column;
+    }
+  }
+  img {
+    grid-column: 1;
+    grid-row: 1/3;
+    @include medium('up') {
+      display: block;
+      width: 70px;
+      margin: 0 auto;
+    }
+  }
+  h4 {
+    font-size: 24px;
```

Comment:
should be using rem to set the font sizes, we globally set the font size for the entire application in _html-tags.scss under responsive